cors.go

```diff
@@ -13,8 +13,7 @@ type Config struct {
 	AllowAllOrigins bool
 
 	// AllowedOrigins is a list of origins a cross-domain request can be executed from.
-	// If the special "*" value is present in the list, all origins will be allowed.
```

Comment:
Yes, support for wildcard after https://github.com/gin-contrib/cors/pull/24 merged. Thanks for your reminder.